### ISSUE TITLE
Adobe Acrobat Error

### ISSUE THREAD

**Issue by curtiskoelling** [signintech/gopdf#64]
When I try to open a generated pdf in Adobe Acrobat I get the following error.

> There was an error processing a page. There was a problem reading this document (135).

I can pm the offending document.

**Comment by curtiskoelling** [comment]
If I run the tool through the validation tool at [pdf-online](https://www.pdf-online.com/osa/validate.aspx) I get the following errors.
```
Validating file "qc_report.pdf" for conformance level pdf1.7
The "endobj" keyword is missing.
The key Subtype is required but missing.
The encoding for character code 0 in font 'Helvetica' is missing.
The encoding for character code 3 in font 'Helvetica' is missing.
The encoding for character code 0 in font 'MyriadPro' is missing.
The encoding for character code 3 in font 'MyriadPro' is missing.
The encoding for character code 20 in font 'MyriadPro' is missing.
The encoding for character code 28 in font 'MyriadPro' is missing.
The encoding for character code 21 in font 'MyriadPro' is missing.
The encoding for character code 19 in font 'MyriadPro' is missing.
The encoding for character code 26 in font 'MyriadPro' is missing.
The encoding for character code 15 in font 'MyriadPro' is missing.
The encoding for character code 17 in font 'MyriadPro' is missing.
The encoding for character code 22 in font 'MyriadPro' is missing.
The encoding for character code 0 in font 'Helvetica' is missing.
The encoding for character code 3 in font 'Helvetica' is missing.
The encoding for character code 0 in font 'MyriadPro' is missing.
The encoding for character code 3 in font 'MyriadPro' is missing.
The encoding for character code 10 in font 'MyriadPro' is missing.
The "endobj" keyword is missing.
The key Subtype is required but missing.
The encoding for character code 1 in font 'Helvetica' is missing.
The encoding for character code 29 in font 'MyriadPro' is missing.
The encoding for character code 19 in font 'MyriadPro' is missing.
The encoding for character code 26 in font 'MyriadPro' is missing.
The encoding for character code 18 in font 'MyriadPro' is missing.
The encoding for character code 20 in font 'MyriadPro' is missing.
The encoding for character code 28 in font 'MyriadPro' is missing.
The encoding for character code 21 in font 'MyriadPro' is missing.
The encoding for character code 24 in font 'MyriadPro' is missing.
The encoding for character code 23 in font 'MyriadPro' is missing.
The encoding for character code 27 in font 'MyriadPro' is missing.
The encoding for character code 16 in font 'MyriadPro' is missing.
The encoding for character code 22 in font 'MyriadPro' is missing.
The encoding for character code 17 in font 'MyriadPro' is missing.
The encoding for character code 25 in font 'MyriadPro' is missing.
The encoding for character code 8 in font 'MyriadPro' is missing.
The encoding for character code 31 in font 'MyriadPro' is missing.
The encoding for character code 15 in font 'MyriadPro' is missing.
The encoding for character code 0 in font 'Helvetica' is missing.
The encoding for character code 3 in font 'Helvetica' is missing.
The encoding for character code 0 in font 'MyriadPro' is missing.
The encoding for character code 3 in font 'MyriadPro' is missing.
The encoding for character code 10 in font 'MyriadPro' is missing.
The encoding for character code 1 in font 'Helvetica' is missing.
The encoding for character code 15 in font 'MyriadPro' is missing.
The encoding for character code 17 in font 'MyriadPro' is missing.
The encoding for character code 21 in font 'MyriadPro' is missing.
The encoding for character code 19 in font 'MyriadPro' is missing.
The encoding for character code 20 in font 'MyriadPro' is missing.
The encoding for character code 26 in font 'MyriadPro' is missing.
The encoding for character code 22 in font 'MyriadPro' is missing.
The document does not conform to the requested standard.
The document doesn't conform to the PDF reference (missing required entries, wrong value types, etc.).
The document contains fonts without embedded font programs or encoding information (CMAPs).
Done.
```

**Comment by curtiskoelling** [comment]
Found the problem.
You cannot supply a font family name to GoPdf.AddTTFFont that has a space in it.

**Comment by oneplus1000** [comment]
fixed a739119dbde0fc47f6e4b4ff10db9408bb06a824
Thank you again for issue report.
